Mathlib/Data/Polynomial/RingDivision.lean

```diff
@@ -76,7 +76,7 @@ theorem modByMonic_eq_of_dvd_sub (hq : q.Monic) {p₁ p₂ : R[X]} (h : q ∣ p
 theorem add_modByMonic (p₁ p₂ : R[X]) : (p₁ + p₂) %ₘ q = p₁ %ₘ q + p₂ %ₘ q := by
   by_cases hq : q.Monic
   · cases' subsingleton_or_nontrivial R with hR hR
-    · simp only [eq_iff_true_of_subsingleton] 
```

Comment:
Please make a separate PR to fix the whitespace in this file, it adds significant noise to this forward-port

Reply:
I'm not sure where this comes from! I didn't touch all those lines.

Reply:
Its vscode auto-truncating the lines. Ideally we'd have a linter than enforced this. I think @mattrobball needs to enable whitespace stripping in their editor!